### ISSUE TITLE
Request: alternate recipe for netherrackdust and tarsand

### ISSUE THREAD

**Issue by ibrokemybanjo** [ReikaKalseki/Reika_Mods_Issues#860]
I'm not sure if you saw my post on the subreddit but they're not going ot unban the grinder so I would like if you added another recipe. Thanks.

**Comment by ReikaKalseki** [comment]
No. Your admins need to not be morons.

**Comment by HyperionNexus** [comment]
If they refuse to unban the grinder then I suggest you leave the server and find another one. You can't just cut out a whole step of mod progression and hope it works.

**Comment by TomeWyrm** [comment]
The "onerous" rules are in place mostly because admins can be idiots. Find another server. Trust me when I say that I don't ever say that likely. I in fact usually roast people on a spit for saying that. But any place where the admin bans things without thinking of the consequences is not worthy of administrating a server.

**Comment by ibrokemybanjo** [comment]
Do you think you could try and convince them? I'll see if i can get them to come here.

**Comment by TomeWyrm** [comment]
I wouldn't. We're not liable to be very nice

**Comment by ReikaKalseki** [comment]
Nor will they be, probably.

**Comment by 1Matthias** [comment]
@ibrokemybanjo If you really, really want to try and stick with this server, (Not the option I recommend), I would be willing to talk to them. I am not able to talk via voice, but I can try to convince them to lift their restrictions.

**Comment by ibrokemybanjo** [comment]
Well, they said there's no chance of them letting you come to our Mumble, so they're going to have to come here.

**Comment by kingoftheblock** [comment]
My time's valuable and every second counts. What do you want?

**Comment by 1Matthias** [comment]
Well, just wanted to let you know that by banning the grinder, that blocks access to 95% of the content in Rotarycraft. RoC uses a progression-based tech tree, with each machine leading to the next. Removing one means that all of the other machines after that machine are no longer accessible. For instance, by banning the grinder in this case, you have removed the main earlygame ore duplication system (Which is only 3x, by the way, and dead slow at that), but also the only way to get ingredients for jet fuel, a major tech tree stepping stone. By removing that, you have removed the way to get bedrock, or a reasonable extractor setup, or pretty much anything else. RoC is very uniquely balanced in this respect.

TL;DR: Blocking the grinder makes almost all of RoC inaccessible, and is technically against the EULA in this case.

**Comment by kingoftheblock** [comment]
Now whose fault is *that*? I count four modders in this discussion and you're seriously telling me that none of you saw this coming? "Uniquely balanced" my ass, it's broken. What kind of idiot mod builds anti-modification DRM in to the very design of the mod? People reporting it to you as an issue? Your problem, not mine. Maybe if you got off your ass and did things properly, this wouldn't happen.

Besides, what's it to you? Who are you to dictate terms to me on how to run my server (well, our server, sorry Legion, but I'm on a roll). Where is the sign that says "Modder's house" at the entrance? Maybe I *want* to stop progression beyond a certain point. Maybe I want to restrict it to certain people, or sell the items in the server shop. Or maybe I just think the damn mod's OP as s*** and would remove it if it weren't so good at attracting those whiny, sniveling players who run off to mommy the second something breaks. "Against the Eula"? What kind of egotistical manchild thinks he has the right to barf out a bunch of lawyerspeak and pretend it's a binding contract?

I'd rather you have come to our enjin, but after the little display here I don't think your welcome.

**Comment by coolsquid** [comment]
**@kingoftheblock: Please put away the hostility. We don't want that around here. Failure to obey by our [rules](https://github.com/ReikaKalseki/Reika_Mods_Issues/blob/master/CONTRIBUTING.md) may result in removal of your posts and/or a ban. This is your only warning.**
> Now whose fault is that? I count four modders in this discussion and you're seriously telling me that none of you saw this coming? "Uniquely balanced" my ass, it's broken. What kind of idiot mod builds anti-modification DRM in to the very design of the mod? People reporting it to you as an issue? Your problem, not mine.

If you dislike the mod so strongly, remove it from your server.
> Maybe if you got off your ass and did things properly (y'know, like everyone else does? Can't be that hard, can it?), this wouldn't happen.

First of all, he's not obligated to "fix" anything. The vast majority of Minecraft mods, including Reika's, are provided on an "AS-IS" basis.

Secondly, how can you "fix" something that works like it's supposed to do?
> Maybe I want to restrict it to certain people, or sell the items in the server shop.

They're not your items to sell.
> Or maybe I just think the damn mod's OP as s*** and would remove it if it weren't so good at attracting those whiny, sniveling players who run off to mommy the second something breaks.

So you're installing the mod just so you can get more players to sell items you don't own to? You sound like a truly nice server admin...
> "Against the Eula"? What kind of egotistical manchild thinks he has the right to barf out a bunch of lawyerspeak and pretend it's a binding contract?

Eh, like 90% of the content providers in the world... Nearly every website, service, or piece of software you use has some form of EULA. Heck, the site you're using right now has one. Examples of "egoistical manchildren" include Bill Gates, Notch, and millions of others.

**Comment by ReikaKalseki** [comment]
> Now whose fault is that?

Everyone aside from you seems to have reached a consensus - you. Even with your needlessly inflated self-worth, you must admit that when four people are openly saying that you are the one at fault, and a fifth is probably only not doing so because they fear retribution, you *just may* want to start considering alternative viewpoints.

> I count four modders in this discussion

Actually, the only relevant one here is me. I am the one who wrote the mod in question. I am the one who spent weeks to months designing the progression system. I am the one who knows better than anyone else how it all works. And I am the one who wrote the rules so as to prevent exactly this outcome - i.e. an ignorant jackass breaking everything because he lacks either or both understanding or interest in the effects of his changes.

> What kind of idiot mod builds anti-modification DRM in to the very design of the mod?

Depending on how you define DRM - besides very loosely - either everyone or noone. There is - despite common accusation - no internal check system or user lock. On the other hand, the mod *is* designed and balanced in such a way that in most cases it only works properly when left unmodified. But that latter category includes many, *many* mods. Are they all idiots too?

> Maybe if you got off your ass and did things properly (y'know, like everyone else does? Can't be that hard, can it?)

Entitlement commentary aside, there is no inherent requirement or even incentive to make any mod behave in any standardized way, and there is nothing wrong with a mod that works differently. It only becomes an issue when someone wants to make changes and is too lazy or incompetent - as for which describes you, take your pick - to learn the nuances of the mod. If I were to log into your server and start whining that I could not start acting like it was a different server (e.g. to break rules specific to your server, or have the same lack of regard for build TPS impact that is common), you would quite rightfully start making judgements about my common sense and/or intellect. Yet you unashamedly and unironically do the same here. Shall I start making the same judgements about you?

> Besides, what's it to you?

Aside from the fact that I am now spending my time dealing with someone who acts like a spoiled toddler, how about the fact that, this being a game, I want people to *enjoy* playing my content, something you directly compromised? Or perhaps the fact that at some point this is probably going to generate spurious bug reports? Or maybe the possibility of this spawning all sorts of rumors, ranging from "RotaryCraft doesn't work" to outright claims of targeted behavior?

> Who are you to dictate terms to me on how to run my server

As stated above, the one who wrote the license to which you agreed. But sure, ignore the rules. Might as well ignore real-world contracts and restrictions as well, right? Are rules not for losers and "little people", after all?

> Maybe I want to restrict it to certain people

Banned by my terms of service. Per-player restriction is inherently unfair and usually used either to give admins or their friends special privileges or outright to - 

> sell the items in the server shop

 - try to make money off the mod. What exactly do you think gives you the privilege of monetizing something you had no part in creating, designing, or maintaining, and indeed to which your only contributions have been destructive? On top of that, even if you are not afraid of me, you might want to be afraid a certain other entity whose regulations you are violating - **Mojang**.

> so good at attracting those whiny, sniveling players who run off to mommy

A great strategy to attract and keep players, that. Lie to them, add mods that only work on the surface, and otherwise wring every playable moment you can so you can start to feel good about yourself. Then go on to denigrate and mock them for daring to be displeased at your mismanagement. I can make it even better, by *preserving your saying this forever*! How fun!

> What kind of egotistical manchild thinks he has the right to barf out a bunch of lawyerspeak and pretend it's a binding contract?

Nearly every software developer on the planet, backed up by dozens of laws in hundreds of countries, including nearly all developed ones.

> after the little display here I don't think your welcome.

Your opening line can be summarized as "keep it quick, you aren't worth much of my time", you whine about having to create an account on the not-exactly-obscure site called GitHub, and then you go off on a rant based on nothing but your own warped reality, and you have the gall to call 1M's reply a "display"?"

**Comment by 1Matthias** [comment]
@kingoftheblock I will answer each of your points in turn. 

> Now whose fault is that? I count four modders in this discussion and you're seriously telling me that none of you saw this coming?

Currently, there are only 2 modders in this discussion, Reika and CoolSquid, and of those 2, Reika is the only RoC dev. I am just kind of here to help.

> "Uniquely balanced" my ass, it's broken. What kind of idiot mod builds anti-modification DRM in to the very design of the mod?

There are different methods of balancing mods. By this logic, Thaumcraft is broken because it does not allow you to remove all wands and continue as if nothing has happened. Also, there are areas of RoC that are modifiable. 

> People reporting it to you as an issue? Your problem, not mine. Maybe if you got off your ass and did things properly (y'know, like everyone else does? Can't be that hard, can it?), this wouldn't happen.

Except that if the progression is a series of rooms, this is like filling one of the rooms with concrete, and then getting annoyed at the architect of the building (Reika) for making it so that filling that room with concrete makes it so that the tenants of the building are annoyed that they can't get at the last room. The fault is not with the architect, but with the person who filled the room with concrete.

> Besides, what's it to you? Who are you to dictate terms to me on how to run my server (well, our server, sorry Legion, but I'm on a roll). Where is the sign that says "Modder's house" at the entrance? Maybe I want to stop progression beyond a certain point.

> Maybe I want to restrict it to certain people, or sell the items in the server shop. Or maybe I just think the damn mod's OP as s*** and would remove it if it weren't so good at attracting those whiny, sniveling players who run off to mommy the second something breaks. 

Actually.... Selling items has been against the Minecraft EULA for a while, and it is technically illegal. You can ask for donations, or sell purely cosmetic items, but selling functional items has been banned for 2 years. 

> "Against the Eula"? What kind of egotistical manchild thinks he has the right to barf out a bunch of lawyerspeak and pretend it's a binding contract?

Well, for one, the EULA *IS* a binding contract. All of them are, whether you read them or not, and they are as real as a mortgage, or a car loan.

**Comment by g4vr0che** [comment]
> Now whose fault is that?

FWIW, as someone casually following this issue with no real stake in either side, yours (kingoftheblock's).

**Comment by TomeWyrm** [comment]
@kingoftheblock Also not a modder, I just help around here to try and take some of the load off of Reika.

**Comment by ibrokemybanjo** [comment]
Please, King, they have a point.

**Comment by LegionMaster400** [comment]
No, they don't. King and I spent hours making sure that our server was properly geared for what we want it to be, and it's not our problem if another mod's so badly designed that it can't fit into that vision. Besides, this server is ours. We pay the hosting fees, we found volunteers to moderate the enjin and the Mumble, and we collect and manage all revenues. You want to have a say in how the server is run, go make your own and see how well it works out for you. If you leave, though, just remember that our policy states that all your server possessions get turned over to us, and all your subscriptions are immediately terminated, so you should think and choose wisely.

@ReikaKalseki 

Frankly, I don't give a flying f*** about what your terms say we can and can't do. Even if I accept your claim that your motives are to preserve fairness and enjoyment (and let's be honest, you haven't managed to pull the wool over that many eyes and it's a well-known fact that the reason so many mods have such rules is because modders are butthurt over not being able to sell their mods while servers rake in the cash and want to bring everyone else down with them) the simple fact is that we can't keep a server alive on good wishes. We need to pay hosting fees, we need to pay for advertising, and we need a safety pool for emergencies like settlements or upcoming rankings. I'm sure as sugar not going to pay that out of my own pocket, and I bet neither King nor our team of volunteers will either, and that means we need an income stream. Selling cosmetics gets you f*** all, but selling mods lines the coffers quite nicely. On top of that, most of our users are probably 15 or younger, and their parents (who are the real source of the money) are statistically more likely to shell out for something that actually means something. I'll agree with you on that Mojang also bans the practice, but I'll also go on to say that they're a bunch of ivory-tower fools for it, and they forget that not everyone has a rich uncle moneybags to foot the bill. This is a business like anything else, it don't run on nice words and praise.

**Comment by g4vr0che** [comment]
Way to go. Break the mod they come to your server to play, then hold them ransom to keep them for leaving. 

And if this doesn't fit your vision of what your server should be, just remove the whole mod. You'll save space, which will make your hosting cheaper in the long run.

Nobody here is butthurt about not being able to sell mods. Believe it or not, many programmers write code because they want to contribute, and many gladly do it for free in the hopes that their software is used widely and enjoyed. That's just how open source works, including the Linux OS that almost certainly runs your server. What makes developers butthurt is when they write something and someone else arbitrarily breaks it and prevents people from using it properly for other people, who then complain to the developer that it's broken, when in reality, it's only broken because someone buttmunch wants it to be.

**Comment by TomeWyrm** [comment]
@LegionMaster400 No, they were tired of the bad publicity, wasted support time, and many other problems caused by people extorting money form parents by exploiting their children for profit.

Mojang holds the stance that any community that can't justify its existence without resorting to exploitation doesn't deserve to use their game to try and do so.

**Comment by ibrokemybanjo** [comment]
Can we please try and find something? I really don't wanna have to move to another server just to try rotarycraft.

**Comment by HyperionNexus** [comment]
You could try Rotarycraft in SP.

**Comment by 1Matthias** [comment]
From the look of it, you are. The admins of the current server you are on have engaged in illegal activity, and when told to stop multiple times, have declined. If you have made any payments, then I would get the transfers reversed, as what they are doing is illegal on multiple counts.

**Comment by TomeWyrm** [comment]
They are likely to be shut down soon. They've admitted to breaking multiple EULA/ToS in a public location. I'd be surprised if they haven't been reported to their hosting company, ISP, and Mojang by this point.

What drew you to this server? I'm betting that we could find you a server that actually follows the rules and has a great community that supports it.

**Comment by kingoftheblock** [comment]
@ibrokemybanjo, If you don't stop whining, you'll **have** to find another server. Shut the f*** up.

**Comment by g4vr0che** [comment]
And now threats. Keep it classy.

@ibrokemybanjo move out from these dictators and find a server that values your presence.

**Comment by TomeWyrm** [comment]
@ibrokemybanjo Is it not obvious that they value you only as a money source? You're nothing but a replaceable cog to them, not a person

**Comment by LegionMaster400** [comment]
@ibrokemybanjo 

https://www.reddit.com/r/feedthebeast/comments/4kvyhm/how_to_get_rotarycraft_jetfuel_without_grinder/

DID YOU SERIOUSLY GO FULLY PUBLIC WITH THIS? DO YOU KNOW WHAT YOU COULD DO? That's it, youre gone. Say goodbye to everything you worked on.
![](http://static.planetminecraft.com/files/resource_media/screenshot/1329/lavafall_5964074.jpg)

I'm having your name added to MCBans as I speak. You won't be able to get on other servers, not with that, so you'll just have to make your own.

**Comment by g4vr0che** [comment]
Wait, you're banning him for telling people you broke mods?

**Comment by TomeWyrm** [comment]
Wow, you're not only an ass, you're slow. Did you not read the first post?

Hey @ibrokemybanjo what is the server's front page? These guys need to be reported to Mojang and their host provider for breaking the Mojang commercial use guidelines

**Comment by kingoftheblock** [comment]
> you'll just have to make your own

Not under our umbrella. I've got friends at MCprohosting and Creeperhost, and you can be darn sure I'm not going to let this just slide.

**Comment by TomeWyrm** [comment]
Do you have friends at Mojang or Microsoft? Because you'll need them soon.

**Comment by TomeWyrm** [comment]
Also, please get ibrokemybanjo added to a public ban site, it will provide a directly and easily traceable trail to you and your server, which is flagrantly breaking the terms of service for the game you're playing.

**Comment by FurryDJ** [comment]
@kingoftheblock and @LegionMaster400  As much as I'd like to carry on this "discussion" I'd highly advise you refrain from harassing @ibrokemybanjo I'm actively contacting representatives from CreeperHost and McProHosting respectively, and am eagerly awaiting their responses on the matter.

**Comment by FurryDJ** [comment]
Update: I've spoken to Aaron Mills one of the team members of CreeperHost and they appear to have never heard of @kingoftheblock or @LegionMaster400 and they don't seem to operate on a global blacklisting basis for petty manners. I am still awaiting official word from MCProHosting but so far it appears these are empty threats. You don't have to worry. @ibrokemybanjo

**Comment by FurryDJ** [comment]
I've just received word from M.r. Eli Levy the head of Partner Relations for McProHosting and he says he has no idea who either of @kingoftheblock or @LegionMaster400 are. That means you have nothing to worry about @ibrokemybanjo These guys are just trying to bully and scare you. Hope you have a nice day and we're happy you enlightened us on this issue, and as for you @kingoftheblock and @LegionMaster400 We recommend you crawl back to your server, fix non-EULA compliant issues and not bring your empty threats back here. Good day.

**Comment by 1Matthias** [comment]
@ibrokemybanjo I sent you a PM on reddit, if you have not seen it, would you mind looking at it and responding? Thank you!

**Comment by gwydion0917** [comment]
@ibrokemybanjo As another mod author and the owner of a community server at a tier 3 data center. You are welcome to our servers. Just let me know if you are interested. @kingoftheblock or @LegionMaster400 can't affect you on my servers. They would find themselves being banned quicker than anything. And we never charge for things on the server.

**Comment by TomeWyrm** [comment]
@gwydion0917 People like you maintain my hope for this community. Thank you for being a decent human being.

**Comment by gwydion0917** [comment]
There is one thing to charge for access in general, some people have to pay for their servers. I am in a position where I am the network admin at the data center, so my hosting is free. I really hate where people run servers with a pay to win theme.

**Comment by zmaster587** [comment]
@ibrokemybanjo  Just going to put in my two cents, I worked as bascially the sole server tech for what was once a large organisation (JAC Gaming), even if they did get you blacklisted on those "global ban" mods or whatever.  They are not often installed on servers.  Not to mention I understand that there are bill to pay, but treating players like cattle is a surefire way to lose all of them.  I speak from experience, that is main reason JAC is only a shell of what it used to be.  I was sick of the attitude and left.  Many left after me for the same reason.  My statement is double edged: Running a server just to make money is not going to get you anywhere, and honestly, staying with a server like that is probably not going to be nearly as fun long term.

@kingoftheblock While I understand servers cost money to run, in order to run a successful server, you cannot treat people as no more than a source of income.  I have seen this type of mentality on many servers before, they never last.  However even under the new EULA you can give out non cosmetic items IF the items are given to all players on the server regardless of if they've spend money.  Something like $5 for everyone to get flight for a day is technically allowed unless I am sorely mistaken.  As the EULA is a binding agreement, I strongly suggest you modify your setup to comply.  But I end with this:  Reika does not owe it to you to do anything with his mod, this is true for all mod devs, most of them do it for fun, other than the top 10 mods, mod devs really don't make much income off of their mods.

**Comment by zerotheliger** [comment]
"Insert clip of shia labeouf clapping"

good show @kingoftheblock you've proved today couple facts for us.

first you are the worse admin i have ever seen to grasp the concept of running a minecraft server possibly expecting to become rich off of it some how... or you just wanted to have alot of power over kids?

second you enjoy torturing people to protect your image even though you have done this all in a public place, which you freaking out makes no sense you kinda dug your own grave there dude.        i don't know how you think this wouldn't get out. also expect some mod authors to be filing for damages since you have been making money without their permission.

third umm well this ain't exactly a fact more of a theory but i have a feeling nobody has ever taught you the concept of treat others how you like to be treated because you are about to learn that real fast buddy :D

**Comment by keybounce** [comment]
>  On top of that, most of our users are probably 15 or younger, and their parents (who are the real source of the money) are statistically more likely to shell out for something that actually means something.

Alright. Does anyone who knows the rules at Visa, Mastercard, or American Express, or heck, Paypal, know the rules about processing illegal transactions over credit cards (or at Paypal) and how to report those?

I suspect that whatever system is being used to accept payments ultimately comes down to a credit card, and most of them prohibit illegal transactions. Since they are admitting to targetting underage children, it gets worse.

I suspect that the credit card company will cut the accounts very very quickly, possibly filing cease and desist letters, if not something stronger, very quickly, and that would shut down the server faster than anything else.

**Comment by Kolatra** [comment]
Just FYI, if this spirals out of control again, the issue will be locked, either by me, Reika, or any of the other moderators.

**Comment by TomeWyrm** [comment]
Actually, I'm going to be preemptive. If someone wants to continue this, create a minecraft forums or reddit post on the subject and let me know (I am on both platforms, same name as here). I'll provide a link to that discussion for people to follow.

**Comment by Kolatra** [comment]
That's a good idea actually. Thank you Tome.

**Comment by freyskol** [comment]
FYI, for future people who read this.. This is NOT what an issue tracker is used for. 8)

**Comment by ReikaKalseki** [comment]
Also for future readers: Both @kingoftheblock and @LegionMaster400's accounts have been purged for abuse, making this discussion somewhat hard to follow. A shortened version has been preserved [here](https://sites.google.com/site/reikasminecraft/things-not-to-say).
